src/extensions/extensionsHelper.ts

```diff
@@ -423,6 +430,32 @@ export async function publishExtensionVersionFromLocalSource(args: {
   );
   validateSpec(subbedSpec);
 
+  if (args.stage !== "stable") {
```

Comment:
This function probably should have had tests already, but as we add more functionality, it needs them more. Can you add a test that checks the behavior when different values are passed for stage?

Reply:
Added tests.